components/gfx/paint_task.rs

```diff
@@ -708,7 +754,7 @@ impl WorkerThread {
 
 enum MsgToWorkerThread {
     Exit,
-    PaintTile(usize, BufferRequest, Option<Box<LayerBuffer>>, Arc<StackingContext>, f32),
+    PaintTile(usize, BufferRequest, Option<Box<LayerBuffer>>, Arc<StackingContext>, f32, bool),
```

Comment:
Please either document what the `bool` does here or do something like `enum LayerMode { Mode2D, Mode3D }` to make it self-documenting.